README.md

```diff
@@ -243,14 +242,10 @@ Below is a diagram of how the files are published in each environment in the `e2
 ```
 . <-- GCS directory set by _PIPELINE_PUBLISH_GCS_PATH
 └── TAG_NAME or GIT COMMIT HASH <-- Git tag used for the release (release.yaml) OR git commit hash (e2e-test.yaml)
-    ├── prediction
-    │   ├── assets
-    │   │   └── some_useful_file.json
-    │   └── prediction.json   <-- compiled prediction pipeline
-    └── training
-        ├── assets
-        │   └── training_task.py
-        └── training.json   <-- compiled training pipeline
+    ├── training.json
+    ├── prediction.json
+    ├── assets
+    │   └── some_useful_file.json
 ```
```

Comment:
@browningjp-datatonic In my view we don't need this detailed overview anymore, rather a description of how the parent folder will be named.

Reply:
not sure what you're suggesting to have instead, but I am happy as long as it is clear! The CI/CD behaviour is not immediately intuitive so it's important that we are very clear in the docs